python/python/bystro/api/auth.py

```diff
@@ -79,7 +79,7 @@ class UserProfile(Struct, rename="camel"):
         The accounts of the user.
     role : str
         The role of the user.
-    last_login : datetime.datetime
+    lastLogin : datetime.datetime
```

Comment:
This is not needed. rename=“camel” takes a lastLogin and stores the value on last_login, so that we don’t need camel case in Python.

Reply:
@dlin30 please change the docstring to match the python property (last_login)